pkg/windows/calico.go

```diff
@@ -60,11 +60,11 @@ users:
 	calicoConfigTemplate = template.Must(template.New("CalicoConfig").Funcs(replaceSlashWin).Parse(`{
   "name": "{{ .Name }}",
   "windows_use_single_network": true,
-  "cniVersion": "{{ .CNI.Version }}",
+  "cniVersion": "0.3.1",
```

Comment:
is it necessary to hard-code the CNI version?

Reply:
No, it is not. I don't know why I thought it was a good idea :). Let me add it as part of the config